scripts/armrunner.sh

```diff
@@ -0,0 +1,30 @@
+###
+# Alternative ARM Runner installer to setup PhotonVision service files and JAR
+# for ARM based builds such as Raspberry Pi, Orange Pi, etc.
+###
+NEW_JAR=$(realpath $(find . -name photonvision\*-linuxarm64.jar))
+echo "Using jar: " $(basename $NEW_JAR)
+
+DEST_PV_LOCATION=/opt/photonvision
+sudo mkdir -p $DEST_PV_LOCATION
+sudo cp $NEW_JAR ${DEST_PV_LOCATION}/photonvision.jar
+
+TARGET_WANTS=/etc/systemd/system/multi-user.target.wants
+sudo mkdir -p $TARGET_WANTS
+cd $TARGET_WANTS
+sudo bash -c "printf \
+\"[Unit]
```

Comment:
This service is missing the loud CPU flag that we comment out by default. Our generator has a known good version of this service file, so we might be able to delete this step entirely

Reply:
That's what I thought too, but wasn't sure. I removed the service file write, and left a comment in the script saying that it assumes the image that arm-runner-action pulls has the servicefile pre-included.